core/templates/domain/exploration/ExplorationObjectFactory.ts

```diff
@@ -52,6 +52,7 @@ import { ExplorationChange } from './exploration-draft.model';
 
 export interface ExplorationBackendDict {
   'auto_tts_enabled'?: boolean;
+  'correctness_feedback_enabled'?: boolean;
```

Comment:
![Screenshot from 2021-05-17 12-03-29](https://user-images.githubusercontent.com/24855641/118447371-521d2900-b70e-11eb-9645-6a18c835c1e7.png)

actual dict returned by backend.

So, correctness_feedback_enabled property is used in exploration-player-state-service. So, added this.